apps/accounts/automations/gather_account/manager.py

```diff
@@ -385,10 +385,11 @@ def do_run(self, *args, **kwargs):
                         ga = ori_account
                         self.update_gathered_account(ori_account, d)
                     ori_found = username in ori_users
+                    # 需要提前创建出来，AccountRisk 外键关联 GatherAccount 记录
+                    self.create_gathered_account.finish()
+                    self.update_gathered_account.finish()
                     risk_analyser.analyse_risk(asset, ga, d, ori_found)
 
-                self.create_gathered_account.finish()
-                self.update_gathered_account.finish()
                 self.update_gather_accounts_status(asset)
                 if not self.is_sync_account:
                     continue
```

Comment:
There is no immediate issue with the provided code snippet; however, some improvements could be suggested:

* In `@staticmethod` decorator of `create_gathered_account()`, you might consider making it non-static so that `self.create_gathered_account()` method can call directly.
   - It would simplify access to `_gather_data` in the static context.

Other than these changes, this section does not show any major irregularities or problems related to coding best practices (e.g., Pythonic style guidelines). However, good programming practice and efficient coding includes using appropriate methods and classes according to the problem scope and complexity.